cogs/Voting.py

```diff
@@ -0,0 +1,354 @@
+    :param results: the results from the vote
+    :return: discord.Embed object to send
+    """
+    embed = discord.Embed(title=f"{vote.title} Results:")
+    for opt, count in results.items():
+        embed.add_field(name=opt.head, value=f"{count} votes", inline=False)
+    if not results:
+        embed.add_field(name="No votes yet!", value="Try giving more time to vote")
+    return embed
+
+
+def create_embed(vote):
+    """
+    Creates an embed of the current vote configuration
+    :return: discord.Embed
+    """
+    embed = discord.Embed(title=vote.title)
+    for x, option in enumerate(vote.options):
+        embed.add_field(name=f"{emote_reference[x]} - {option.head}", value=option.body, inline=False)
+    return embed
+
+
+async def get_results(bot, vote):
+    """
+    Gathers the results from all users who were sent the vote
+    :param vote:
+    :param bot: the discord.commands.Bot that sent out the vote messages
+    :return: dict of results
+    """
+    results = {}
+    for u_id, msg_id in vote.sent_to.items():
+        user = bot.get_user(u_id)
+        msg = await user.fetch_message(msg_id)
+        for reaction in msg.reactions:
+            if reaction.count > 1:
+                opt = vote.options[emote_reference[reaction.emoji]]
+                if opt in results.keys():
+                    results[opt] += 1
+                else:
+                    results[opt] = 1
+                break
+    return results
+
+
+class Voting(commands.Cog, name="Vote"):
+    def __init__(self, bot, db_manager=None):
+        """
+        discord cog to manage the voting interface
+        :param bot: the bot that the cog is being run on
+        :param db_manager: a database manager (allows testing on a clean database)
+        """
+        self.bot = bot
+        self.vote_manager = VoteManager()
+        if not db_manager:
+            self.DBManager = KoalaDBManager.KoalaDBManager(KoalaBot.DATABASE_PATH, KoalaBot.DB_KEY)
+            self.DBManager.insert_extension("Vote", 0, True, True)
+
+    @commands.Cog.listener()
+    async def on_raw_reaction_add(self, payload):
+        """
+        Listens for when a reaction is added to a message
+        :param payload: payload of data about the reaction
+        """
+        await self.update_vote_message(payload)
+
+    @commands.Cog.listener()
+    async def on_raw_reaction_remove(self, payload):
+        """
+        Listens for when a reaction is removed from a message
+        :param payload: payload of data about the reaction
+        """
+        await self.update_vote_message(payload)
+
+    @commands.group(name="vote")
+    async def vote(self, ctx):
+        """
+        Use k!vote create <title> to create a vote!
+        """
+        if ctx.invoked_subcommand is None:
+            await ctx.send(f"Please use `{KoalaBot.COMMAND_PREFIX}help vote` for more information")
+
+    # @commands.check(KoalaBot.is_admin)
+    @vote.command(name="create")
```

Comment:
```suggestion
    @vote.command(name="create")
    @commands.check(KoalaBot .vote_is_enabled)
```
Check that voting is enabled